src/styles/table.styl

```diff
@@ -119,6 +120,17 @@
 
 mime-types('fil-file-', '../assets/icons/')
 
+.fil-file-preview
+    content ''
+    position absolute
+    left 0
+    top 50%
+    transform translateY(-50%)
```

Comment:
What bothers you ? the `transform` ? Because it's the best solution for centering elements in CSS except flexbox but it requires a heavier structure. So for single element centering, it's still the best method (and even more in this case) to this day.